sections/design/design.tex

```diff
@@ -1,10 +1,15 @@
 \chapter{Design}
-\input{sections/design/why-react.tex}
-\input{sections/design/introduction-to-react.tex}
+In the following chapter we examine the design of this system.
+To do so, we will design and describe prototypes for the system to be used when designing the front end.
+A class diagram will be constructed to define the parts of the system, as well as an architecture diagram to define how the different components interact.
+A choice of framework has to be made for the front end, and this will be described in this chapter, and an introduction to the chosen framework will be written.
+Finally, we discuss the API design, and the structure of the database through an entity-relationship diagram.
 \input{sections/design/prototypes.tex}
 \input{sections/design/class-diagram.tex}
 \input{sections/design/architecture-diagram.tex}
+\input{sections/design/why-spa.tex}
+\input{sections/design/why-react.tex}
+\input{sections/design/introduction-to-react.tex}
```

Comment:
Det virker møj fint, men du må gerne lige kigge hurtigt på why react og introduction to react.

I why react skriver vi at React er et javascript framework, og det gentager vi i introduction til react.

Vi kan evt også fyre en sætning ind i why react at react er godt til SPA, så der er react en fordel.

Reply:
har fjernet den der gentagelse og ændret lidt, ved ikke om der skal være flere ændringer, tror det er fint bortset fra vi måske skal udvide why react hvis vi får tid